hil/client/node.py

```diff
@@ -26,11 +26,11 @@ def register(self, node, subtype, *args):
         # Node requires which OBM, and knows arguments required
         # for successful node registration.
 
-        obm_api = "http://schema.massopencloud.org/haas/v0/obm/"
-        obm_types = ["ipmi", "mock"]
         # FIXME: In future obm_types should be dynamically fetched.
         # We need a new api call for querying available
         # and currently active drivers for HIL
+        # obm_api = "http://schema.massopencloud.org/haas/v0/obm/"
+        # obm_types = ["ipmi", "mock"]
```

Comment:
off topic note: I read this the whole comment here, and now we have an API to get active drivers, so I guess we can implement this now (in a separate PR of course).